### PR TITLE
Change the UI and Visual Appearance of Dashboard and Login Page [ Issue #68 ]

### PR DESCRIPTION
**Pull request by itsme-shivamkumar**
## Changes Proposed:
- Move About Page Button: The About page button has been successfully relocated from the footer to the Navbar, improving navigation for our users.
- Background Update: The web app's background has been changed to incorporate a gradient with animation, adding a visually appealing element and enhancing the overall design.
- Dashboard Styling: Various styling modifications have been made to the dashboard, improving its visual appeal and providing a more polished and enjoyable user experience.

## Before and After Preview:
- Before: 

https://github.com/srivastavaritik/Extraspace-Cloud/assets/91745441/4bc91bbd-23b3-43c9-812e-7f5be3a42a41

-After:

https://github.com/srivastavaritik/Extraspace-Cloud/assets/91745441/60061242-7856-4860-8a6d-484c162efc60

### I kindly request a mentor to review and merge these improvements into the main repository. I am eager to discuss any further improvements or suggestions to enhance the user interface of our web application.

### CODE REVIEW

**Comment by itsme-shivamkumar** [comment]
> Someone is attempting to deploy a commit to a Personal Account owned by @srivastavaritik on [Vercel](https://vercel.com).
> 
> @srivastavaritik first needs to [authorize it](https://vercel.com/git/authorize?type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%22433a1fcc2a5dfd1aa7eaba93c928e3542850bf34%22%7D%2C%22id%22%3A%22QmVvWu8MvyEZzWv54h5gceL1PL6DjeXB4FiZszEapcBjhB%22%2C%22org%22%3A%22srivastavaritik%22%2C%22prId%22%3A78%2C%22repo%22%3A%22Extraspace-Cloud%22%7D).

I thought I had to deploy that so I attempted that, I hope that's not a problem since I'm new to contribution and did that unintentionally.🥲

**Comment by srivastavaritik** [comment]
> > Someone is attempting to deploy a commit to a Personal Account owned by @srivastavaritik on [Vercel](https://vercel.com).
> > @srivastavaritik first needs to [authorize it](https://vercel.com/git/authorize?type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%22433a1fcc2a5dfd1aa7eaba93c928e3542850bf34%22%7D%2C%22id%22%3A%22QmVvWu8MvyEZzWv54h5gceL1PL6DjeXB4FiZszEapcBjhB%22%2C%22org%22%3A%22srivastavaritik%22%2C%22prId%22%3A78%2C%22repo%22%3A%22Extraspace-Cloud%22%7D).
> 
> I thought I had to deploy that so I attempted that, I hope that's not a problem since I'm new to contribution and did that unintentionally.🥲

It's totally fine, I'll review this and probably merge or give suggestions on this tomorrow.

**Comment by srivastavaritik** [comment]
@AKD-01 please review and approve

**Comment by srivastavaritik** [comment]
There are some changes that you should do for successfully merging this

File: src/components/drive/Dashboard.js [hunk 1]
```diff
@@ -41,13 +41,15 @@ export default function Dashboard() {
...
+                width: "120px",
+                textAlign:"center",
+                padding: " .2rem .2rem .2rem .2rem",
+                borderRadius: "2px",
                 backgroundColor: "#c2d2df5e",
+                marginLeft:"10px"
               }}
             >
-              Folders
+              {"> "}Folders
```
Comment (srivastavaritik) [inline src/components/drive/Dashboard.js:52]: do same {"> "} for `Files` and `Links` also

File: src/components/drive/FolderBreadcrums.css [hunk 2]
```diff
@@ -10,6 +10,5 @@ padding: .2rem;
     color: rgb(163, 214, 255);
 }
 .bgcrum{
-    color: aliceblue;
-    background-color: #0000009c;
+    
 }
```
Comment (srivastavaritik) [inline src/components/drive/FolderBreadcrums.css:14]: remove unwanted styles

**Review by srivastavaritik — COMMENTED** [review]
please do these changes at earliest @itsme-shivamkumar

**Comment by srivastavaritik** [comment]
and please raise another pr

File: src/components/drive/Dashboard.js (hunk 1 above)
Comment (itsme-shivamkumar) [reply src/components/drive/Dashboard.js:52]: Sure👍

**Comment by itsme-shivamkumar** [comment]
> please do these changes at earliest @itsme-shivamkumar

Sure👍

**Comment by itsme-shivamkumar** [comment]
> and please raise another pr

Sure, I have created an issue #111 Kindly assign me so that I can raise a PR for the above changes. Thank You.

File: src/components/drive/FolderBreadcrums.css (hunk 2 above)
Comment (itsme-shivamkumar) [reply src/components/drive/FolderBreadcrums.css:14]: Sure I have done it

**Pull request merged**